conda_forge_tick/auto_tick.xsh

```diff
@@ -66,8 +66,17 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
                                    fork=fork,
                                    gh=gh)
 
-    # migrate the `meta.yaml`
     recipe_dir = os.path.join(feedstock_dir, 'recipe')
+    # if postscript/activate no noarch
+    if migrator.__class__.__name__ == 'Noarch' and any(
+            x in os.listdir(recipe_dir) for x in [
+                '{}-pre-unlink'.format(attrs['feedstock_name']),
+                '{}-pre-link'.format(attrs['feedstock_name']),
+                '{}-post-link'.format(attrs['feedstock_name']),
```

Comment:
Do these scripts always follow this naming convention? I think I've seen pre-link.sh/.bat before.